packages/polaris-common/src/common-methods.ts

```diff
@@ -1,6 +1,10 @@
-export const runAndMeasureTime = async (runnable: () => {}) => {
+export async function runAndMeasureTime(runnable: () => {}) {
```

Comment:
maybe put this file inside utils directory, and modify the file name to utils-methods